src/instructlab/sdg/pipeline.py

```diff
@@ -78,6 +84,33 @@ def batching_enabled(self) -> bool:
         """
         return self.batch_size > 0 and self.batch_num_workers != 1
 
+    @property  # type: ignore
+    def client(self):
+        return self.clients.get(self.DEFAULT_CLIENT_KEY, None)
```

Comment:
A few questions:
1. Should we also handle the case where `self.client = None` but `self.clients` contains a single client not set to the default client? It seems like in this case, we could perhaps make the assumption that this _is_ the default client. Otherwise, if there are multiple clients then it would become ambiguous. 
2. Why do we specify `None` as the explicit default here? Won't this be the default of the `.get` method?